number_trivia_app/linux/flutter/generated_plugin_registrant.cc

```diff
@@ -0,0 +1,11 @@
+//
```

Comment:
Please add all files from flutter folder to .gitignore and delete these new files

Reply:
This was not done as expected